k8s/install/services/cluster-autoscaler/Makefile

```diff
@@ -0,0 +1,18 @@
+export AUTOSCALER_CPU_LIMIT ?= 100m
+export AUTOSCALER_MEMORY_LIMIT ?= 300Mi
+export AWS_REGION ?= us-west-2
+export RENDER_POD_CMD ?= j2 autoscaler.yaml.j2 | kubectl apply -f -
+export AUTOSCALER_IMAGE ?= k8s.gcr.io/cluster-autoscaler:v1.3.0
```

Comment:
I believe you need `v1.1.x`
https://github.com/kubernetes/autoscaler/tree/master/cluster-autoscaler#releases

(not to be confused with the [version of the cluster](https://github.com/kubernetes/autoscaler/tree/master/cluster-autoscaler/cloudprovider/aws#kubernetes-version))

Reply:
You are correct, and adjusted to 1.1.3